### PR TITLE
adding image compression info to README

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per conversations between @peterbe and myself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -451,7 +451,7 @@ a new image to the `files/en-us/web/css` document.
     cp /some/path/my-cool-image.png files/en-us/web/css/
     ```
 
-1. Run our `filecheck` command on each image you add. It'll complain if something's wrong.
+1. Run our `filecheck` command on each image you add. It'll complain if something's wrong (see also the [Compressing images](####Compressing_images) section below).
```
Comment (peterbe) [inline README.md:454]: I know it was already there but I don't like this nomenclature of "our". It makes it feel like there's a them and us. 
Can we sneak in a change to say "Run the `filecheck` command ..."?

Comment (peterbe) [inline README.md:454]:
```suggestion
1. Run the `filecheck` command on each image you add. It'll complain if something's wrong 
  (see also the [Compressing images](#compressing-images) section below).
```

**Review by peterbe — APPROVED** [review]
Approved by but the link wasn't working.

**Comment by chrisdavidmills** [comment]
Agreed, and I also changed another similar instance of "our" to "the".

**Pull request merged**